apps/frontend/src/app/2_molecules/AssetAmountInput/AssetAmountInput.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { FC, useCallback } from 'react';
+
+import { AmountInput, Paragraph, Select, SelectOption } from '@sovryn/ui';
+import { Decimal, Decimalish } from '@sovryn/utils';
+
+import { BOB_CHAIN_ID } from '../../../config/chains';
+
+import { AmountRenderer } from '../AmountRenderer/AmountRenderer';
+import { AssetRenderer } from '../AssetRenderer/AssetRenderer';
+import { MaxButton } from '../MaxButton/MaxButton';
+
+type AssetAmountInputProps = {
+  label?: string;
+  maxAmount?: Decimalish;
+  invalid?: boolean;
+  amountLabel?: string;
+  amountValue?: string | number;
+  onAmountChange?: (value: string) => void;
+  assetValue: string;
+  assetUsdValue?: Decimalish;
+  assetOptions: SelectOption<string>[];
+  onAssetChange: (asset: string) => void;
+};
+
+export const AssetAmountInput: FC<AssetAmountInputProps> = ({
+  label,
+  maxAmount,
+  amountLabel,
+  amountValue,
+  onAmountChange,
+  invalid,
+  assetValue,
+  assetUsdValue,
+  assetOptions,
+  onAssetChange,
+}) => {
+  const assetOptionRenderer = useCallback(
+    ({ value }) => (
+      <AssetRenderer
+        chainId={BOB_CHAIN_ID}
```

Comment:
Maybe this should be a param. In molecules this is a generic component, not just for aave